Dockerfile.ubuntu

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:20.04
+# VOLUME [ "/app" ]
+WORKDIR /app
+RUN apt update && apt upgrade
+RUN apt install -y build-essential
+RUN apt install -y curl
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+COPY . .
+CMD ["~/.cargo/bin/cargo"]
```

Comment:
Does `CMD ["cargo"]` not work?

Reply:
For some reason no

Reply:
Hmm, interesting. I think that might be because the rust installer doesn't source the package. Instead of installing `curl` and using the script, can you try doing `apt install -y cargo` instead? It's in the official repos and that should add it to the PATH.

Reply:
Sweet yeah that did the trick